assets/stylesheets/application/components/_apply.scss

```diff
@@ -2,7 +2,13 @@
 
 .site-c-apply {
   overflow: auto;
-  width: 100%;
-  min-height: 2500px;
+  min-height: 195em; /* mobile first - ~3120px @font-size=16*/
```

Comment:
@TomNUSDS on my localhost (MacOS, current Chrome) the desktop height is working out to ≈ 2560px. The min height listed here was 2500, but currently on the live site the iframe is set to `style=height: 2867px;"` inline on the iframe tag itself.

We'll probably need to bump the em values here a little more to get things just past 2867px on desktop (and similarly for tablet/phone too).

Reply:
_ALSO_ ... oddly ... that inline style is not present on this build of the site but wasn't something in these changes. Maybe that inline style on the iframe tag got removed somewhere along the line and _that's_ where this issue popped up?

Reply:
Oh... removing `width: 100%` is likely a mistake.

I wonder... In the past, I got bit by this issue where the chrome debugger can have this "mapping the source file" feature enabled where any edits you make to test things in the debugger can become actual edits to the source file.